random.js

```diff
@@ -0,0 +1,23 @@
+app.use(express.json())
+
+app.all('/*', (req, res, next) => {
+    res.status(404).send({message: 'Not Found'})
+})
+
```

Comment:
we need to rename this file 😫 - we can have this test with the other tests and then have it as a error handling file. However we still don't need app.use (express.json() and we're not using these error handlers yet